### DISCUSSION TITLE
Help I’m stuck using Diary App - no longer working

### DISCUSSION THREAD

**Discussion by JaneReidDiary** [community/community#126963] [Other Feature Feedback, Questions, & Ideas]
Yesterday l pasted in some text to a diary entry to export it to pdf into an external app. It was large (61K words) but it never worked so I cancelled and exited the app. Today I am unable to load or open the Diary app at all. It just says processing entries with the donut of death and never moves from that opening screenshot with diary logo so I am unable to log an issue. Please help and offer some solutions. My Diary goes back to 2018 and I would hate to lose my entries to-date.

**Comment by ebndev** [comment]
Hi @JaneReidDiary, thanks for being a part of the GitHub Community!

It sounds like a problem with the app itself, so you might want to try asking for help somewhere that focuses on this or seeing if the Diary App has a support page. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
